lua/ttt2/libraries/gameloop.lua

```diff
@@ -446,12 +446,12 @@ if SERVER then
             ---
             -- @realm server
             -- stylua: ignore
-            local win = hook.Run("TTT2PreWinChecker", preventWin)
+            local win = hook.Run("TTT2PreWinChecker", preventWin) -- FIXME: What do we want to send to the hooks here? the convar value? that would always be true. Do we even want to send anything here?
 
             ---
             -- @realm server
             -- stylua: ignore
-            win = win or hook.Run("TTTCheckForWin", preventWin)
+            win = win or hook.Run("TTTCheckForWin", preventWin) -- FIXME: see above
```

Comment:
I'm not sure - isn't prevent win the convar that always prevents a win? What speaks against passing that? It can be true or false. The main TTT2 wincheck, where this convar is used, is also inside this hook

Reply:
jup exactly and this will always be `false` at this point because of:

```lua
        if gameloop.GetRoundState() ~= ROUND_ACTIVE or cvPreventWin:GetBool() then
            return
        end
```
a few lines above.

I'd like to simply remove this, any objections?

Reply:
I looked a bit through the code and you are right. I suspect I accidentally added in my gameloop refactor. Maybe I wanted to do something with it, but then scrapped the idea but forgot to add the parameters.

Because it wasn't in the code before my refactor:

https://github.com/TTT-2/TTT2/blob/aa961a58382d77e169dd02eff48dbd66f216e142/gamemodes/terrortown/gamemode/server/sv_main.lua#L793-L817

Also the hook, that is called here, still doesn't accept this value:

https://github.com/TTT-2/TTT2/blob/cc837c42f79254c4c28f1fb1f378bec7b3a5c2b8/gamemodes/terrortown/gamemode/server/sv_main.lua#L848-L854

Reply:
removed in [7ed4347](https://github.com/TTT-2/TTT2/pull/1574/commits/7ed4347fafa536cdf05d130afe3b4dabd48c199e)